test/v1/connectors/UnignoreEmploymentConnectorSpec.scala

```diff
@@ -40,6 +43,22 @@ class UnignoreEmploymentConnectorSpec extends ConnectorSpec {
         result shouldBe expectedOutcome
       }
     }
+
+    "return the expected response for a HIP request" when {
+      "a valid request is made" in new HipTest with Test with ConnectorTest {
+        def taxYear: TaxYear = TaxYear.fromMtd("2023-24")
```

Comment:
Please remove this in both tests and make change `def taxYear: TaxYear` in Test trait on line 65 `val taxYear: TaxYear = TaxYear.fromMtd("2023-24")` as only 2023-24 tax year is being tested. Also, please ensure to apply these changes in V2, thanks